src/components/SelectionStyles.tsx

```diff
@@ -0,0 +1,60 @@
+/*!
+ * The contents of this file are subject to the Common Public Attribution License Version 1.0 (the “License”); you may not use this file except in compliance with the License. You may obtain a copy of the License at https://mpapp-public.gitlab.io/manuscripts-frontend/LICENSE. The License is based on the Mozilla Public License Version 1.1 but Sections 14 and 15 have been added to cover use of software over a computer network and provide for limited attribution for the Original Developer. In addition, Exhibit A has been modified to be consistent with Exhibit B.
+ *
+ * Software distributed under the License is distributed on an “AS IS” basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the specific language governing rights and limitations under the License.
+ *
+ * The Original Code is manuscripts-frontend.
+ *
+ * The Original Developer is the Initial Developer. The Initial Developer of the Original Code is Atypon Systems LLC.
+ *
+ * All portions of the code written by Atypon Systems LLC are Copyright (c) 2024 Atypon Systems LLC. All Rights Reserved.
+ */
+import React, { useEffect } from 'react'
+
+import { scrollIntoViewIfNeeded } from '../lib/scroll'
+import { useStore } from '../store'
+
+export const SelectionStyles: React.FC = () => {
+  const [{ selectedSuggestionID }, dispatch] = useStore((state) => ({
+    selectedSuggestionID: state.selectedSuggestionID,
+  }))
+
+  useEffect(() => {
+    document.addEventListener('click', (e) => {
```

Comment:
If we use editor state to track this, then every node with a change will have to be re-rendered every time this state changes (i.e. on every click). This might not be an issue for most nodes, but it is for nodes like like `bibliography_element` for which the rendering is expensive. We can add logic to improve the performance for these cases, but I think that's too complicated just to achieve some style change.